common/s2n-codec/src/decoder/checked_range.rs

```diff
@@ -76,3 +77,25 @@ impl fmt::Debug for CheckedRange {
         write!(f, "{}..{}", self.start, self.end)
     }
 }
+/// The following harnesses were automatically generated by an
+/// experimental tool developed by the Kani team.
+#[cfg(test)]
+mod bolero_harnesses {
+    use super::*;
+    #[test]
+    #[cfg_attr(kani, kani::proof)]
+    fn bolero_test_25_len() {
+        bolero::check!()
+            .with_type()
+            .cloned()
+            .for_each(|callee: CheckedRange| Some(callee.len()));
+    }
+    #[test]
```

Comment:
space above

Reply:
sorry i should have been more clear. i meant all of the new items being added should have a space between them. I'm actually surprised rustfmt doesn't complain about this.